lib/apiservers/engine/backends/backends.go

```diff
@@ -92,44 +93,89 @@ func Init(portLayerAddr, product string, config *config.VirtualContainerHostConf
 	// the vic-machine installer timeout will intervene if this blocks for too long
 	pingPortLayer()
 
+	if err := hydrateCaches(); err != nil {
+		return err
+	}
+
 	log.Info("Creating image store")
 	if err := createImageStore(); err != nil {
 		log.Errorf("Failed to create image store")
 		return err
 	}
 
+	serviceOptions := registry.ServiceOptions{}
+	for _, registry := range insecureRegs {
+		insecureRegistries = append(insecureRegistries, registry.Path)
```

Comment:
Ignore if we're logging this somewhere else, but a log of which registries are added would be useful.  Really anywhere where we set up hosts, paths, etc during startup is useful to know during runtime.